### DISCUSSION TITLE
Additional explanation needed!

### DISCUSSION THREAD

**Discussion by Tezei** [Cyfrin/foundry-full-course-cu#2706] [Ideas]
Hello,
Thank you for awesome training and explanations.

I want to suggest a little improvement on this video - https://updraft.cyfrin.io/courses/foundry/foundry-simple-storage/create-new-foundry-setup

When a user uses Windows like me there is no previous explanation of using commands in VS Code, i.e. "forge init". I am getting "Unknown command". And it is normal. Because Foundry is installed not on Windows, but on WSL.

So, I am suggesting to include in this video suggestion to activate WSL in a VS Code terminal first and then go to "mnt" folder and then to c:\ or else and run "forge init" in a project folder. Or using another technique in your opinion, but definitely additional instructions are needed on this stage.

Also, when I tried to compile directly from VS Code on Windows I got "forge not found" and when I tried to compile outside in the directory I got "Error (6275): Source "src/Counter.sol" not found: File not found. "

**Important:** Also, when we are restarting VS Code we received following error "Couldn't find forge binary..." So, maybe it is now suitable to use WSL for Windows users.

Best regards!

**Comment by EngrPips** [comment]
Hello @Tezei, Did you mistakenly open this issue twice?

**Reply by Tezei** [reply]
yes, sorry. I will move it to Discord as you suggested!

**Reply by EngrPips** [reply]
It's fine.
